src/fesom_module.F90

```diff
@@ -147,8 +163,13 @@ subroutine fesom_init(fesom_total_nsteps)
 #endif
 
 #if defined (__oasis)
-
-        call cpl_oasis3mct_init(f%partit,f%partit%MPI_COMM_FESOM)
+!        call cpl_oasis3mct_init(f%partit,f%partit%MPI_COMM_FESOM)
+! kh 02.12.21
+#if defined(__recom) && defined(__usetp)
```

Comment:
"localCommunicator" is neither defined in fesom_module nor in MOD_PARTIT. Should this really be "localCommunicator" or "f%partit%MPI_COMM_FESOM"?

Reply:
I can't really answer this. These lines already exited in fesom2.5-recom and another earlier version where Özgür and I merged their complex recom version and my simpler paleo version. But I can't find them in my paleo version where Kai first time implemented tracer parallelisation. There this part of code is still in fvom_main.f90 and only contains:
#if defined (__oasis)

! kh 21.03.22 pass num_fesom_groups to coupler
    call cpl_oasis3mct_init(MPI_COMM_FESOM, num_fesom_groups)
#endif
    t1 = MPI_Wtime()

    call par_init
......

So I think it should be for the case with tracer parallelisation:
call cpl_oasis3mct_init(f%partit, f%partit%MPI_COMM_FESOM, num_fesom_groups)